lib/api/apiUtils/authorization/prepareRequestContexts.js

```diff
@@ -154,13 +154,6 @@ function prepareRequestContexts(apiMethod, request, sourceBucket,
                 generateRequestContext('objectPutTaggingVersion');
             requestContexts.push(putObjectVersionRequestContext);
         }
```

Comment:
why are we changing permissions?
if it was (already) a bug, it shoudl be fixed on 8.8 as well instead here...

Reply:
@benzekrimaha did you miss this one?

Reply:
it's not changing the permission ,this code is actually duplicated see here : https://github.com/scality/cloudserver/pull/5711/files#diff-465d8f1849b67c1a40847391c0a7ea074b6357a8b7cdeb2db2be89ad9dfce50bL79-L80

we are never going to cross this condition